src/service/cli/generate.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+const fs = require(`fs`);
+
+const {
+  getRandomInt,
+  shuffle
+} = require('../utils');
+
+const DEFAULT_COUNT = 1;
+const FILE_NAME = `mocks.json`;
+
+const TITLES = [
+  `Продам книги Стивена Кинга`,
+  `Продам новую приставку Sony Playstation 5`,
+  `Продам отличную подборку фильмов на VHS`,
+  `Куплю антиквариат`,
+  `Куплю породистого кота`,
+];
+
+const SENTENCES = [
+  `Товар в отличном состоянии.`,
+  `Пользовались бережно и только по большим праздникам.`,
+  `Продаю с болью в сердце...`,
+  `Бонусом отдам все аксессуары.`,
+  `Даю недельную гарантию.`,
+  `Если товар не понравится — верну всё до последней копейки.`,
+  `Это настоящая находка для коллекционера!`,
+  `Если найдёте дешевле — сброшу цену.`,
+  `Таких предложений больше нет!`,
+  `При покупке с меня бесплатная доставка в черте города.`,
+];
+
+const CATEGORIES = [
+  `Книги`,
+  `Разное`,
+  `Посуда`,
+  `Игры`,
+  `Животные`,
+  `Журналы`,
+];
+
+const OfferType = {
+  offer: `offer`,
+  sale: `sale`,
+};
+
+const SumRestrict = {
+  min: 1000,
+  max: 100000,
+};
+
+const PictureRestrict = {
+  min: 1,
+  max: 16,
+};
+
+const getPictureFileName = (pictureNumber) => pictureNumber > 10 ? `item${pictureNumber}.jpg` : `item0${pictureNumber}.jpg`;
+
+const generateOffers = (count) => (
+  Array(count).fill({}).map(() => ({
```

Comment:
```suggestion
  Array(count).map(() => ({
```
ибо `.fill({})` тут лишнее

Reply:
Как оказалось, .fill нелишний. map не сработает, если не указать, чем заполнить новый массив. Array(number) создаст пустой массив (вообще без элементов) со свойством length, равным number. Соответственно map не по чему итерироваться.
https://itnext.io/heres-why-mapping-a-constructed-array-doesn-t-work-in-javascript-f1195138615a

Reply:
Что то я профакапил этот момент